thirdparty/build-thirdparty.sh

```diff
@@ -1545,12 +1545,43 @@ build_concurrentqueue() {
     cp ./*.h "${TP_INSTALL_DIR}/include/"
 }
 
+#clucene
+build_clucene() {
+    if [[ -z ${USE_AVX2} ]]; then
+        USE_AVX2=1
+    fi
+    if [[ -z ${BUILD_TYPE} ]]; then
+        BUILD_TYPE=Release
+    fi
```

Comment:
This block can be omitted. Just pass `-DCMAKE_BUILD_TYPE=RelWithDebInfo` to CMake.

Reply:
Only with RelWithDebInfo flags is not enough, CLucene also need to compile ASAN edition,

Reply:
> Only with RelWithDebInfo flags is not enough, CLucene also need to compile ASAN edition,

It is a third-party library, we can just use it. Why do we need ASAN build?

Reply:
It's not just a third-party lib, we added a lot features into it, sometimes we need ASAN to debug problems

Reply:
> It's not just a third-party lib, we added a lot features into it, sometimes we need ASAN to debug problems

Okey.